spec/transactions-spec.coffee

```diff
@@ -59,7 +59,7 @@ describe "Ledger transactions view", ->
       advanceClock(@editor.getBuffer().stoppedChangingDelay)
 
       markers = @editor.findMarkers()
-      expect(markers.length).toEqual 11
+      expect(markers.length).toEqual 22
```

Comment:
Not sure about this. Seems odd (but this was the original assertion value). This fails when I run it locally, but maybe it's a rendering difference between Linux and OSX.